doc/nrf/includes/sample_board_rows.txt

```diff
@@ -97,3 +97,7 @@
 | :ref:`nRF5340 DK <ug_nrf5340>` | PCA10095 | :ref:`nrf5340dk_nrf5340 <nrf5340dk_nrf5340>` | ``nrf5340dk_nrf5340_cpuapp`` |
 |                                |          |                                              |                              |
 |                                |          |                                              | ``nrf5340dk_nrf5340_cpunet`` |
+
+.. nrf21540dk_nrf52840
+
+| :ref:`nRF21540 DK <ug_nrf52>` | PCA10112 | :ref:`nrf21540dk_nrf52840 <nrf21540dk_nrf52840>` | ``nrf21540dk_nrf52840`` |
```

Comment:
I think it's misleading to link to ug_nrf52 here ("Working with nRF52 Series"), because there are no mentions of the nRF21540 DK in there. It's probably better to link to ug_radio_fem ("Radio Front-end Module (FEM) support") instead - but does that contain the required information?

Reply:
What about updating the `ug_nrf52` with the mention to `nrf21540dk_nrf52840`?